templates/bin_setup.erb

```diff
@@ -8,7 +8,7 @@ set -e
 
 # Set up Ruby dependencies via Bundler
 gem install bundler --conservative
-bundle check || bundle install
+bundle check || bundle install --path=vendor/bundle
```

Comment:
As part of a step before install, could we configure the bundler path? http://bundler.io/v1.3/man/bundle-config.1.html

Failing that, travis sets `ENV["CI"]` which we can detect and set the path option here if necessary.